sdk/identity/identity/src/msal/utils.ts

```diff
@@ -110,6 +111,27 @@ export const defaultLoggerCallback: (
     }
   };
 
+/**
+ * @internal
+ */
+export function convertToMSALLogLevel(logLevel: AzureLogLevel| undefined): msalCommon.LogLevel | undefined{
+ if(logLevel)
+  switch(logLevel){
+  case "error":
+    return msalCommon.LogLevel.Error;
+  case "info":
+    return msalCommon.LogLevel.Info;
+  case "verbose":
+    return msalCommon.LogLevel.Verbose;
+  case "warning":
+    return msalCommon.LogLevel.Warning;
+ }
+ else{
+  return undefined
```

Comment:
Does undefined mean "use whatever default MSAL uses"? I think that should be ok then (it'll be Info)

Reply:
no that means it's not set. Do you want me to keep the default as "Info" ?

Reply:
@bgavrilMS Also we don't have an equivalent for "trace". We have another package @azure/core-tracing which enables the tracing within the credentials